_tests/integration/ios_test.go

```diff
@@ -204,6 +250,13 @@ configs:
               - test_repetition_mode: retry_on_failure
               - cache_level: none
           - deploy-to-bitrise-io@%s: {}
+        test_without_building:
+          steps:
+          - pull-intermediate-files@%s: {}
+          - xcode-test-without-building@%s:
+              inputs:
+              - only_testing: $BITRISE_TEST_SHARDS_PATH/$BITRISE_IO_PARALLEL_INDEX
+              - destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
```

Comment:
Let's remove the destination input and rely on the step's default input value. We need to update this from time to time and it is enough if we need to update it at one place only.